src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -307,15 +309,19 @@ function ActivitySummary(params: {
   const isOrder = type === ActivityType.ORDER
 
   if (isOrder) {
-    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind } = activity as Order
+    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind, fulfillmentTime } =
+      activity as Order
 
     const sellAmt = CurrencyAmount.fromRawAmount(inputToken, sellAmount.toString())
     const feeAmt = CurrencyAmount.fromRawAmount(inputToken, feeAmount.toString())
     const outputAmount = CurrencyAmount.fromRawAmount(outputToken, buyAmount.toString())
+    const limitPrice = formatSmart(getLimitPrice(activity as Order))
 
     orderSummary.from = `${formatSmart(sellAmt.add(feeAmt))} ${sellAmt.currency.symbol}`
     orderSummary.to = `${formatSmart(outputAmount)} ${outputAmount.currency.symbol}`
+    orderSummary.limitPrice = `${limitPrice} ${outputAmount.currency.symbol} per ${sellAmt.currency.symbol} `
     orderSummary.validTo = new Date((validTo as number) * 1000).toLocaleString()
+    orderSummary.fulfillmentTime = fulfillmentTime ? new Date(fulfillmentTime).toLocaleString() : undefined
```

Comment:
Question (I can answer with a few clicks but I am lazy and also good for more people because I'm sure this was/will be asked): isn't `fulfillmentTime` a timestamp? Thus also needing to be multiplied by 1000?

I guess the answer is no and it's an ISO date time string, but in any case, might be worth to leave a comment for future reviewers.

Reply:
Yes good you double checked. When I initially logged the 'activity' element, the `fulfillmentTime` returned as a string with a ISO date format. Whereas `validTo` returned as an actual timestamp.